gym_solo/core/obs.py

```diff
@@ -140,9 +142,17 @@ def get_obs(self) -> Tuple[solo_types.obs, List[str]]:
     all_labels = []
     
     for obs in self._observations:
-      all_obs.append(obs.compute())
       all_labels.append(obs.labels)
 
+      values = obs.compute()
+      if self._normalize:
+        a = np.array(values)
+        low = obs.observation_space.low
+        hi = obs.observation_space.high
+        values = (2 * (a - low)) / (hi - low) - 1
```

Comment:
It's a minor comment but add an extra parenthesis for clarity? 
`values = ((2 * (a - low)) / (hi - low)) - 1`

Reply:
done